themes/SuiteP/css/bootstrap.min.css

```diff
@@ -1764,13 +1764,13 @@ pre code {
     width: 58.33333333%}
 
 .col-sm-6 {
-    width: 50%}
+    }
 
 .col-sm-5 {
     width: 41.66666667%}
 
 .col-sm-4 {
-    width: 33.33333333%}
+    width: 16.5%}
```

Comment:
Editing the base twitter bootstrap file is bad practice, as all other col-sm-4 elements will be affected with this change.